clash-vexriscv/src/ffi/impl.cpp

```diff
@@ -51,11 +53,23 @@ VVexRiscv* vexr_init()
 {
 	contextp = new VerilatedContext;
 	VVexRiscv *v = new VVexRiscv(contextp);
-	Verilated::traceEverOn(true);
 	v->clk = false;
 	return v;
 }
 
+VerilatedVcdC* vexr_init_vcd(VVexRiscv *top, const char* path)
+{
+	if (path != NULL){
+		VerilatedVcdC* vcd = new VerilatedVcdC;
+		Verilated::traceEverOn(true);
+		top->trace(vcd, 99);
```

Comment:
What does the 99 stand for? Some kind of depth or length? Would be nice to have a comment here

Reply:
Yes, depth. Though I think there's only one level for VexRisc. Will add a comment.